### ISSUE TITLE
Feature Request: Show Dictated Text

### ISSUE THREAD

**Issue by TheBlueKingLP** [Muirey03/SmallSiri#21]
Please add an option that allow Small Siri to show the text that it dictated.

**Comment by ziwzih** [comment]
Yes please!

**Comment by fgegner** [comment]
Yes that would clean it up and look like more than a glitch on the screen, plus it would be nice to confirm that Siri understood our request
